luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -63,18 +64,20 @@ for weaponDefID, wDef in ipairs(WeaponDefs) do
 	end
 	
 	--generate list of exempted weapons to improve performance
+	local minImpulseFactor = 0.15
+	local minImpulseToDamageRatio = 0.2
```

Comment:
These should be with the other config variables at the top of the file

Reply:
ok